parsons/newmode/newmode.py

```diff
@@ -617,19 +717,21 @@ def get_submissions(self, campaign_id, params=None):
         if params is None:
             params = {}
         params = {"action": campaign_id}
-        response = self.converted_request(endpoint="submission", method="GET", params=params)
+        response = self.converted_request(
+            endpoint="submission", method="GET", params=params, data_key=RESPONSE_DATA_KEY
```

Comment:
I am curious why some calls to `converted_request` pass in `data_key=RESPONSE_DATA_KEY` and some don't